app/controllers/v0/account_controls_controller.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module V0
+  class AccountControlsController < ApplicationController
+    rescue_from ActiveRecord::RecordNotFound, with: :not_found
+    skip_before_action :authenticate, :verify_authenticity_token
+    before_action :authenticate_service_account, :validate_account_control_params
+
+    VALID_CSP_TYPES = %w[logingov idme dslogon mhv].freeze
+
+    def csp_lock
+      user_verification.update!(locked: true)
```

Comment:
should this be rescued and render an error if it fails? you could also just use `update` which will return true or false 

```ruby
if user_verification.update(locked: true)
  Rails.logger.info('UserAccount CSP Lock', serialized_response)
   render json: { data: serialized_response }
else
 # log error
 # render error response
end
  
```